tests/DwitTech.AccountService.WebApi.Tests/Controllers/UserControllerTest.cs

```diff
@@ -353,5 +353,81 @@ public async Task Logout_Should_Return_False_If_AuthorizationHeader_Is_Not_Given
             Assert.IsType<BadRequestObjectResult>(result);
 
         }
+
+        [Fact]
+        public async Task DeleteUser_Returns_NoContentResult()
+        {
+            // Arrange
+            var userId = 1;
+            var _activationService = new Mock<IActivationService>();
+            var iloggerMock = new Mock<ILogger<UserController>>();
+            var _mockAuthService = new Mock<IAuthenticationService>();
+            var _mockUserService = new Mock<IUserService>();
+
+            var userController = new UserController(_activationService.Object, _mockAuthService.Object, _mockUserService.Object, iloggerMock.Object);
+
+            // Act
+            var result = await userController.DeleteUser(userId);
+
+            // Assert
```

Comment:
How is this test successful without you mocking the delete method in the service?

Reply:
> How is this test successful without you setting up the delete method in the service?

Resolved